release-notes/9.0/supported-os.json

```diff
@@ -193,16 +185,15 @@
                         "arm"
                     ],
                     "supported-cycles": [
-                        "20.04",
+                        "24.04",
                         "22.04",
-                        "23.10",
-                        "24.04"
+                        "20.04"
                     ]
                 }
             ]
         },
         {
-            "family": "Windows",
+            "name": "Windows",
             "distributions": [
                 {
                     "name": "Nano Server",
```

Comment:
`Windows Server Core` has the `id` set to `windows-server-core` but `Nano Server` has no `id` at all right now. I will ask the people of `endoflife.date` if there is intention to add those as well.

Reply:
I try to add `windows-nano-server` through https://github.com/endoflife-date/endoflife.date/pull/5323. So we should use that.